src/WCSimRootEvent.cc

```diff
@@ -410,17 +410,23 @@ WCSimRootTrack::WCSimRootTrack(Int_t ipnu,
 
 //_____________________________________________________________________________
 
-WCSimRootCherenkovHit *WCSimRootTrigger::AddCherenkovHit(Int_t tubeID, Int_t mPMTID, Int_t mPMT_PMTID, std::vector<Float_t> truetime,std::vector<Int_t> primParID)
+WCSimRootCherenkovHit *WCSimRootTrigger::AddCherenkovHit(Int_t tubeID, Int_t mPMTID, Int_t mPMT_PMTID, std::vector<Float_t> truetime,
```

Comment:
Just noticing `mPMT` here, so not applicable to HK (regular PMTs) I guess? This part of the code has already diverged from HK version? Hopefully this can be synchronized later, so we should keep track of it.

Reply:
Yes, there's a lot of code specific to mPMTs throughout the nuPRISM fork (not new to this PR). I think there's been discussion of merging for years, and would be really good to do, but it's a very big job given how far they've diverged.